TECH-OVERSIGHT-COMMITTEE.md

```diff
@@ -2,9 +2,14 @@
 
 The Istio Technical Oversight Committee (TOC) is responsible for cross-cutting product and design decisions.
 
+We encourage community members to observe the committee meetings, but we request that all questions and comments
```

Comment:
See if we can make this a bit more friendly:

Community members are encouraged to suggest topics for discussion ahead of the TOC meeting, and are invited to observe the TOC meetings and engage with the TOC during the community feedback period at the end of each meeting.

I expect we'll have to police people for a few meetings before a standard practice gets established.